newIDE/app/src/MainFrame/index.js

```diff
@@ -1611,6 +1611,7 @@ class MainFrame extends React.Component<Props, State> {
 
   _onEditorTabActive = (editorTab: EditorTab) => {
     this.updateToolbar();
+    editorTab.editorRef.performRefresh();
```

Comment:
nitpicking: can we call this `forceUpdateEditor`?

"forceUpdate" is a hint at the forceUpdate from React. Refresh has a slightly different meaning (refresh would be something like making network requests to "refresh" your profile).

Reply:
Let's also add here a comment why we're doing this:
`// Ensure the editors shown on screen are updated. This is for example useful if global objects have been updated in another editor.`

Reply:
Will do that straight away! :)